cardano-addresses.cabal

```diff
@@ -42,6 +42,7 @@ library
   exposed-modules:
       Cardano.AddressDerivation
       Cardano.AddressDerivation.Byron
+      Cardano.AddressDerivation.Cbor
```

Comment:
Feels weird to have this module here because, it's not specific of "AddressDerivation". What about something like:

```
Cardano.Codec.Cbor
```

?

Reply:
will change as you request